charts/tempo/templates/ingester/poddisruptionbudget-ingester.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if gt (int .Values.ingester.replicas) 1 }}
 {{ $dict := dict "ctx" . "component" "ingester" "memberlist" true }}
-apiVersion: {{ include "tempo.pdb.apiVersion" . }}
```

Comment:
is community chart incorrect

Reply:
I don't know, but without changing api version, the chart does not pass the validate-templates